src/plugins/navigation/public/top_nav_menu/top_nav_menu.tsx

```diff
@@ -172,7 +172,7 @@ export function TopNavMenu(props: TopNavMenuProps): ReactElement | null {
             return (
               <>
                 <MountPointPortal setMountPoint={setMenuMountPoint}>
-                  <EuiFlexGroup alignItems="stretch" gutterSize="s">
+                  <EuiFlexGroup alignItems="stretch" gutterSize="none">
```

Comment:
Will it impact the case when the toggle is turned off?

Reply:
Nah. this is within the switch.